umap/static/umap/js/umap.js

```diff
@@ -518,11 +518,11 @@ U.Map = L.Map.extend({
 
   initShortcuts: function () {
     const globalShortcuts = function (e) {
-      const key = e.keyCode,
-        modifierKey = e.ctrlKey || e.metaKey
+      const key = e.keyCode
+      const hasModifier = (e.ctrlKey || e.metaKey) && !e.shiftKey
```

Comment:
What about shortcuts [with a shift key](https://github.com/umap-project/umap/blob/master/umap/static/umap/js/umap.core.js#L326-L329)?

Reply:
Unless I'm blind, which I'm close to, those are not managed here (in the global shortcuts), they are listened on click on some elements, isn't it ?